cypress/support/commands.js

```diff
@@ -54,9 +55,9 @@ Cypress.Commands.add('createArticle', (title, description, body) => {
       url: '/api/articles',
       body: {
         article: {
-          title,
-          description,
-          body,
+          title: faker.random.word(),
```

Comment:
you have inserted here faker data, but in your test file, you again generated the article and then gave generated data to function createArticle(). remove faker data here or do not generate new article data in your test file

Reply:
Fixed